site/app/templates/admin/NotebookBuilder.twig

```diff
@@ -44,6 +44,6 @@
     <div id="notebook-builder">
         <p>Loading...</p>
     </div>
-    <button onclick="javascript:document.getElementById('holder').innerHTML=JSON.stringify(notebook_builder.getJSON(), null, 2)">Show JSON</button>
+    <button class="btn btn-primary btn-nav btn-nav-submit" onclick="javascript:document.getElementById('holder').innerHTML=JSON.stringify(notebook_builder.getJSON(), null, 2)">Show JSON</button>
```

Comment:
I saw the screenshot, and I don't think the buttons should be blue. 
Blue buttons are for important things.
I would say, other than `Save` they do not deserve `btn-primary` (blue button)

Reply:
Then, which color should I use it for non-primary button. Is there any style guide or anything that I can follow?